src/bun.js/bindings/BindgenNodeErrors.cpp

```diff
@@ -0,0 +1,22 @@
+#include "BindgenNodeErrors.h"
+#include "ErrorCode.h"
+
+JSC::EncodedJSValue throwNodeInvalidArgTypeErrorForBindgen(JSC::ThrowScope& throwScope, JSC::JSGlobalObject* globalObject, const WTF::ASCIILiteral& arg_name, const WTF::ASCIILiteral& expected_type, JSC::JSValue val_actual_value)
+{
+    auto actual_value = Bun::JSValueToStringSafe(globalObject, val_actual_value);
```

Comment:
haven't seen this function, is handling Symbol what makes it safe?

Reply:
it is what ErrorCode.cpp has

![image](https://github.com/user-attachments/assets/09e23b03-cc82-4bcf-9351-eb999ae24f30)